src/domain/entities/MetadataEntities.ts

```diff
@@ -0,0 +1,3613 @@
+import { Ref } from "./Ref";
+import { Access, Expression, Id, Style, Translation } from "./Schemas";
+
```

Comment:
We don't need all the entities, even we don't need entities here, just that we know they inherit a ShareableObject which inherits a Ref object